opencensus/trace/ext/django/middleware.py

```diff
@@ -155,8 +173,13 @@ def process_request(self, request):
 
         try:
             # Start tracing this request
-            header = get_django_header()
-            span_context = self.propagator.from_header(header)
+            if isinstance(self.propagator,
```

Comment:
After we combine the `get_django_headers` and `get_django_header`, we don't need the if else here any more. And the `propagator.from_carrier` could be renamed to use `from_header`.